src/chrome/content/rules/Scala-lang.org.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+	Nonfunctional hosts in *.scala-lang.org:
+		- codereview.scala-lang.org (404)
```

Comment:
It's not clear what's the issue is here. It should probably be changed to something such as `Different content HTTP/HTTPS`